src/frontend/komponenter/ManuellJournalfør/KnyttTilNyBehandling.tsx

```diff
@@ -23,7 +23,7 @@ export const KnyttTilNyBehandling: React.FC = () => {
     const { skjema, minimalFagsak, kanKnytteJournalpostTilBehandling } = useManuellJournalfør();
     const { knyttTilNyBehandling, behandlingstype, behandlingstema } = skjema.felter;
     return (
-        <Fieldset legend="Knytt til ny behandling">
+        <Fieldset legend="Knytt til ny behandling" hideLegend>
             <Heading size={'small'} level={'2'}>
                 Knytt til ny behandling
             </Heading>
```

Comment:
Kunne kanskje ha vurdert å ha synlig legend og fjerne Heading rett under - hvis det ser ok ut visuelt

Reply:
Testet det, men størrelsen er feil 😞 da må jeg isåfall endre alle overskrifter på journalføringen så de matcher med hverandre